src/include/CMakeLists.txt

```diff
@@ -55,6 +55,22 @@ configure_file (
 	"${CMAKE_CURRENT_BINARY_DIR}/kdbversion.h"
 	)
 
+file (MAKE_DIRECTORY ${CMAKE_BINARY_DIR}include/)
+configure_file (
```

Comment:
Please write a proposal (or at least a placeholder for a proposal so that we do not forget)